apps/dh/api-dh/source/DataHub.WebApi.Tests/TestServices/GraphQLTestService.cs

```diff
@@ -60,6 +61,7 @@ public GraphQLTestService()
         MeasurementsResponseMapperMock = new Mock<IMeasurementsResponseMapper>();
         RequestSignatureFactoryMock = new Mock<IRequestSignatureFactory>();
         HttpClientFactoryMock = new Mock<IHttpClientFactory>();
+        SettlementReportsClientMock = new Mock<ISettlementReportsClient>();
```

Comment:
suggestion: delete the other client named "SettlementReportClientMock", since we now mock the wrapper client instead.